lms/static/js/spec/courseware/proctored_exams_spec.js

```diff
@@ -1,73 +0,0 @@
-define(['jquery', 'backbone', 'common/js/spec_helpers/template_helpers', 'js/courseware/base/models/proctored_exam_model', 'js/courseware/base/views/proctored_exam_view'
```

Comment:
@chrisndodge I am going to remove that spec file, since we don't need it here anymore.